server_src/lib/dataFunctions/printerClean.js

```diff
@@ -1735,21 +1735,7 @@ class PrinterClean {
     return versionArray;
   }
 }
+
 module.exports = {
   PrinterClean,
 };
-
-//Hacky database check due to shoddy layout of code...
-const mongoose = require("mongoose");
```

Comment:
Moved to `index.js` as I dont like interval code to run when I require a functional file :)
Should be pretty harmless